L/libgeotiff/build_tarballs.jl

```diff
@@ -50,11 +47,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("JpegTurbo_jll"; compat="3.0.1"),
+    Dependency("JpegTurbo_jll"; compat="3.1.1"),
     Dependency("LibCURL_jll"; compat="7.73,8"),
-    Dependency("Libtiff_jll"; compat="4.7"),
-    Dependency("PROJ_jll"; compat="902.500"),
-    Dependency("Zlib_jll"; compat="1.2.12"),
+    Dependency("Libtiff_jll"; compat="4.7.1"),
+    Dependency("PROJ_jll"; compat="902.500.100"),
+    Dependency("Zlib_jll"; compat="1.3.1"),
```

Comment:
This won't work in any released version of Julia: https://github.com/JuliaLang/julia/blob/v1.11.3/stdlib/Zlib_jll/Project.toml#L3

Reply:
What's wrong with 1.2.12?